ephemeralkey.go

```diff
@@ -46,7 +46,7 @@ func (e *EphemeralKey) UnmarshalJSON(data []byte) error {
 		*e = EphemeralKey(ee)
 	}
 
-	e.RawJSON = data
+	e.RawJSON = append(e.RawJSON[:0], data...)
```

Comment:
Would you mind adding a comment above this line to the effect of:

```
// Go does guarantee the longevity of `data`, so copy when assigning `RawJSON`
```

Reply:
Thanks for your review. I added a comment.